nifi-framework-api/src/main/java/org/apache/nifi/cluster/ConnectionState.java

```diff
@@ -24,41 +24,6 @@ public enum ConnectionState {
      */
     NOT_CLUSTERED,
 
-    /**
-     * Instance is in the process of connecting to the cluster
-     */
-    CONNECTING,
-
-    /**
-     * Instance is connected to the cluster
-     */
-    CONNECTED,
-
-    /**
-     * Instance is in the process of disconnecting from the cluster
-     */
-    DISCONNECTING,
-
-    /**
-     * Instance is disconnected from the cluster
-     */
-    DISCONNECTED,
-
-    /**
-     * Instance is offloading
-     */
-    OFFLOADING,
-
-    /**
-     * Instances has completed offloading
-     */
-    OFFLOADED,
-
-    /**
-     * Instance has been removed from the cluster
-     */
-    REMOVED,
-
```

Comment:
This looks valid and seems like it should be reverted.

Reply:
There is now a NodeConnectionState that has those.  These 'ConnectionState' entries look defunct at least from code scanning.     How are you finding they are valid?

Reply:
Yes, in this case, the `StandardClusterDetailsFactory` reads the `NodeConnectionState` and then converts that string name to this `ConnectionState` value.